tests/integration/pascaldekloe.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.20
+
+RUN git clone https://github.com/pascaldekloe/redis.git
+WORKDIR redis
+
+ENV TEST_REDIS_ADDR=localhost
```

Comment:
Thanks!

have you run this image against Redis/DF on the localhost?
it won't work with regular settings - I think you need `172.17.0.1` to access a port outside a container

Reply:
This image ran as -network host with the alpha docker image together went well.